components/playground/playground.go

```diff
@@ -1468,3 +1471,55 @@ func logIfErr(err error) {
 		fmt.Println(err)
 	}
 }
+
+// Check the MySQL Client version
+//
+// Since v8.1.0 `--comments` is the default, so we don't need to specify it.
+// Without `--comments` the MySQL client strips TiDB specific comments
+// like `/*T![clustered_index] CLUSTERED */`
+//
+// This returns `mysql --comments` for older versions or in case we failed to check
+// the version for any reason as `mysql --comments` is the safe option.
+// For newer MySQL versions it returns just `mysql`.
+//
+// For MariaDB versions of the MySQL Client it is expected to return `mysql --comments`.
+func mysqlCommand() (cmd string) {
+	cmd = "mysql --comments"
+	mysqlVerOutput, err := exec.Command("mysql", "--version").Output()
+	if err != nil {
+		return
+	}
+	vMaj, vMin, _, err := parseMysqlCommand(string(mysqlVerOutput))
+	if err == nil {
+		if vMaj == 8 && vMin >= 1 { // 8.1.0 and newer
+			return "mysql"
+		}
+	}
+	return
+}
+
+// parseMysqlCommand parses the output from `mysql --version` that is in `versionOutput`
+// and returns the major, minor and patch version.
+//
+// New format example: `mysql  Ver 8.2.0 for Linux on x86_64 (MySQL Community Server - GPL)`
+// Old format example: `mysql  Ver 14.14 Distrib 5.7.36, for linux-glibc2.12 (x86_64) using  EditLine wrapper`
+func parseMysqlCommand(versionOutput string) (vMaj int, vMin int, vPatch int, err error) {
```

Comment:
Maybe Version instead of Command?
```suggestion
func parseMysqlVersion(versionOutput string) (vMaj int, vMin int, vPatch int, err error) {
```

Reply:
Addressed in 03faa76fe16c2f09c6377ede6b4635ffaf44c7c5